apps/new/widget/lib/Projects.jsx

```diff
@@ -0,0 +1,79 @@
+const fetchProjects = (props) => {
+  const app = props.app || "${alias_old}";
+  const type = props.type || "project";
```

Comment:
it's not a component, so we should rename the file to 'projects` instead of `Projects`.
what do you think?

Reply:
sure!